aws_lambda_powertools/metrics/provider/datadog/datadog.py

```diff
@@ -65,6 +65,27 @@ def __init__(
         )
         self.default_tags = default_tags or {}
         self.flush_to_log = resolve_env_var_choice(choice=flush_to_log, env=os.getenv(constants.DATADOG_FLUSH_TO_LOG))
+        self.metrics_disabled = self.is_metrics_disabled()
+
+    @staticmethod
+    def is_metrics_disabled() -> bool:
```

Comment:
Hi @anafalcao, how about moving this function to the `functions.py` file in the `metrics` folder?